src/js/_enqueues/wp/wp-image-mime-fallback-loader.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @output wp-includes/js/wp-image-mime-fallback-loader.js
+ */
+
+( function( document, image_src, settings ) {
+
+	var image = document.createElement( 'image' );
+	image.src = image_src + 'jIAAABXRUJQVlA4ICYAAACyAgCdASoCAAEALmk0mk0iIiIiIgBoSygABc6zbAAA/v56QAAAAA==';
+	image.onload = function() {
+		image.onload = undefined;
+		image.src = image_src + 'h4AAABXRUJQVlA4TBEAAAAvAQAAAAfQ//73v/+BiOh/AAA=';
+	};
+	image.onerror = function() {
+		var script_src = settings.imagemimefallback || {};
```

Comment:
I don't think it makes sense here to fall back to an empty object, it's also unnecessary. In addition, see my other comment regarding usage of camel-case for JS properties.
```suggestion
		var script_src = settings.imageMimeFallbackScript;
```